sdk/tables/aztable/models.go

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package aztable
+
+// QueryOptions contains a group of parameters for the Table.Query method.
+type ListOptions struct {
+	// OData filter expression.
+	Filter *string
+	// Specifies the media type for the response.
+	Format *OdataMetadataFormat
+	// Select expression using OData notation. Limits the columns on each record to just those requested, e.g. "$select=PolicyAssignmentId, ResourceId".
+	Select *string
+	// Maximum number of records to return.
+	Top *int32
+}
+
+func (l *ListOptions) toQueryOptions() *QueryOptions {
+	if l == nil {
+		return &QueryOptions{}
```

Comment:
Can we just get rid of QueryOptions entirely and just use ListOptions?
Is QueryOptions code generated?

Reply:
QueryOptions is generated code, in the future we can look at changing the name of QueryOptions post generation and get rid of the one in models.go